### DISCUSSION TITLE
The panel cannot be edited

### DISCUSSION THREAD

**Discussion by FangPengbo** [grafana/grafana#58203] [Dashboards, Panels & Visualizations]
I have admin permission, but I can't edit the panel.

![a](https://raw.githubusercontent.com/FangPengbo/leetcode-daily/main/Snipaste_2022-11-04_12-01-20.png)

Can someone help me? Thank you

**Comment by torkelo** [comment]
is it a repeated panel ?

**Reply by FangPengbo** [reply]
yes,I think I know the problem. Thank you>

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
